packages/web-components/src/components/card-group/__stories__/card-group.stories.ts

```diff
@@ -243,7 +247,7 @@ export default {
       CardGroup: ({ groupId }) => ({
         cards: Array.from({
           length: number('Number of cards', 5, {}, groupId),
-        }).map(() => cardRandomPhrase()),
+        }).map(() => cardRandomPhrase(inPercy())),
```

Comment:
Is it necessary to pass this as an argument? Can't you just use it in the method itself?

Reply:
Yeah, I can do that. I'll change it right away.

Reply:
Done 👍